### PR TITLE
Upgrade to Ember 3

### PR DESCRIPTION
**Pull request by scottkidder**
All but one test passing.

Would appreciate anyone taking a look.

### CODE REVIEW

**Comment by knownasilya** [comment]
@dguettler travis would help with getting PRs merged quicker 👍

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Lauren Elizabeth Tan
+Copyright (c) 2018 Lauren Elizabeth Tan
```
Comment (ro0gr) [inline LICENSE.md]: This is a classical `ember-cli-update` issue with unintentional update of a copyright year :laughing: 
I think it should be reverted

**Comment by scottkidder** [comment]
After merging #140 all tests are passing.

Thanks @knownasilya

File: package.json [hunk 2]
```diff
@@ -45,6 +45,7 @@
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^4.0.0",
+    "ember-router-service-polyfill": "^1.0.3",
```
Comment (knownasilya) [inline package.json]: 👍

**Comment by scottkidder** [comment]
Even after adding the router-service polyfill the same test that was fixed by #140 is now failing only in lts-2.12

Is it time to drop support for 2.12 yet?

**Comment by scottkidder** [comment]
Hi @dguettler any idea when you may be able to review and get this merged?

**Comment by scottkidder** [comment]
*ping* @dguettler

**Comment by geoffreyd** [comment]
Ember 2.12 support ended in June 2018 (https://www.emberjs.com/blog/2017/04/29/ember-2-13-released.html), so seems like it's reasonable to drop support. 
If support is being dropped, then 2.12 should be removed from ember-try/CI configs.

Would this be a breaking change requiring a major version bump?

**Pull request merged**